opcua/server/address_space.py

```diff
@@ -515,7 +515,7 @@ def set_attribute_value(self, nodeid, attr, value):
 
         for k, v in cbs:
             try:
-                v(k, value)
+                v(k, value, old)
```

Comment:
Hi, sorry for taking that long to respond, I was working on another aspect of the project. I was thinking about this now, and, if I were to create a new class, say, "Monitored Item Values" to store current and old value, I was thinking of adding it to the MonitoredItemData class. Do you think this could be a problem? Is this class defined in the specfication? I took a quick look and didn't find anything.